### PR TITLE
feat: injected hack code for setting font size in rich text

### PR DESCRIPTION
**Pull request by LofWalter**
feat: injected hack code for setting font size in rich text

### CODE REVIEW

File: plugins/format/plugin.js [hunk 1]
```diff
@@ -85,35 +85,43 @@ CKEDITOR.plugins.add( 'format', {
 					inputNode.focus();
```
Comment (dfguo) [inline plugins/format/plugin.js:85]: this commit needs a better commit message

File: core/selection.js [hunk 2]
```diff
@@ -1735,7 +1735,7 @@
 				this.selectRanges( ranges );
 				this.lock();
 				// Return to the previously focused element.
-				focused && !focused.equals( this.root ) && focused.focus();
```
Comment (dfguo) [inline core/selection.js:1738]: need a comment to explain why it's been commented out

File: plugins/format/plugin.js (hunk 1 above)
Comment (LofWalter) [reply plugins/format/plugin.js:85]: fixed

File: core/selection.js (hunk 2 above)
Comment (LofWalter) [reply core/selection.js:1738]: fixed

**Pull request merged**